syntax = "proto3";

service Showtime {
    rpc GetSchedule(Empty) returns (AllSchedule) {}
    rpc GetMoviesByDate(Date) returns (Schedule) {}
}

message Schedule {
    string date = 1;
    repeated string movies = 2;
}

message AllSchedule {
    repeated Schedule schedules = 1;
}
message Date{
    string date = 1;
}

message Empty {

}